pkgs/by-name/ch/chirpstack-concentratord/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  gitUpdater,
+  protobuf,
+  libloragw-2g4,
+  libloragw-sx1301,
+  libloragw-sx1302,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "chirpstack-concentratord";
+  version = "4.4.2";
+
+  src = fetchFromGitHub {
+    owner = "chirpstack";
+    repo = "chirpstack-concentratord";
+    rev = "v${version}";
+    hash = "sha256-UbUtNJuz8zfhHzyOiT/mRNtNRmdoNnuszrVSbLoVGK8=";
+  };
+
+  cargoHash = "sha256-JXIyrbBRGJR9mjvawU46mBfGYxZPpYl9aB9k3WBA/co=";
+
+  buildInputs = [
+    libloragw-2g4
+    libloragw-sx1301
+    libloragw-sx1302
+  ];
+
+  nativeBuildInputs = [
+    protobuf
+    rustPlatform.bindgenHook
+  ];
+
+  passthru.updateScript = gitUpdater { rev-prefix = "v"; };
+
+  meta = {
+    description = "Concentrator HAL daemon for LoRa gateways";
+    homepage = "https://www.chirpstack.io/";
+    license = lib.licenses.mit;
+    maintainers = [ lib.maintainers.stv0g ];
+    platforms = [ "aarch64-linux" ];
```

Comment:
Why is this only available on aarch64-linux?

Reply:
Specific hardware?

Reply:
Its a LoRaWAN gateway which I have been testing on a ARM64 single-board computer.
But I checked, that in fact its just using Linux's SPI/I2C drivers.
So it also works on other architectures.

I tested the build now on:
- aarch64-linux
- x86_64-linux

Fixed in 1ec52f0b996b